src/app/user-list/user-list.component.html

```diff
@@ -1,4 +1,11 @@
-<h6 class="mt-3">user-list.component</h6>
-<app-user-single></app-user-single>
-<app-user-single></app-user-single>
-<app-user-single></app-user-single>
+
+<div id="user-list">
+  <h6 class="mt-3 text-center fw-bold">user-list.component</h6>
+  <ol>
+    <li class="mt-3" *ngFor = "let objs of obj">
```

Comment:
l'ngFor non vuole spazi, inoltre l'array dovrebbe essere plurale e il singolo elemento dell'array invece dovrebbe essere singolare
*ngFor="let obj of objs"